DockerSetup.md

```diff
@@ -0,0 +1,78 @@
+# Local DP Dash development on Docker
+
+This document outlines the set up for the development environment using docker
+
+## Requirements
+
+- [Complete all the steps up to the initialize step](./README-PNL.md#initialize)
+
+## Prepare the Configs:
+
+- Copy the contents from the dashboard directory.
+- Create a directory called configs inside the server directory of the project.
+- Open the basePathConfig.js and replace it's contents with this.
+
+```js
+const basePathConfig = ''
+
+export default basePathConfig
+```
+
+- Comment lines 32-35. It should look like this.
```

Comment:
what file are these lines in?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/358#issuecomment-1669999988